src/Pages/SignUp/SignUp.js

```diff
@@ -1 +1,153 @@
-import React from "react";
+import React, { Component } from "react";
+import "./SignUp.scss";
+import { serverIp } from "../../config";
+import SignUpNav from "./SignUpNav";
+
+class Signup extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      first_name: "",
+      last_name: "",
+      birthday: "",
+      email: "",
+      password: "",
+      confirm_password: "",
+      button1: false,
+    };
+  }
+
+  goToLogin = () => {
+    this.props.history.push("/login");
+  };
+
+  handleInputs = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  handleButton = () => {
+    this.setState({
+      button1: !this.state.button1,
+    });
+  };
+
+  okClickHandler = (e) => {
+    fetch(`${serverIp}/user/signin`, {
+      method: "POST",
+      body: JSON.stringify({
+        first_name: this.state.first_name,
+        last_name: this.state.last_name,
+        birthday: this.state.birthday,
+        email: this.state.email,
+        password: this.state.password,
+        check_password: this.state.check_password,
```

Comment:
```suggestion
  okClickHandler = (e) => {
    const { first_name, last_name, birthdat, email, password, check_password } = this.state
    fetch(`${serverIp}/user/signin`, {
      method: "POST",
      body: JSON.stringify({
        first_name,
        last_name,
        birthday,
        email,
        password,
        check_password,
```
구조 분해 할당과 객체 단축 속성명을 활용하면 이런 식으로 줄여 쓸 수 있습니다

Reply:
네, 반영하겠습니다.